### PR TITLE
Feature/tags improvement

### PR DESCRIPTION
**Pull request by zaporozhetsAnton**

### CODE REVIEW

File: src/components/Tags/index.tsx [hunk 1]
```diff
@@ -23,9 +24,13 @@ export interface ITagsProps {
   style?: any;
   error?: string;
   onChange?: (selectedTags: (string | number)[]) => any;
+  onRef?: (arg: any) => void;
```
Comment (pavel-stryber) [inline src/components/Tags/index.tsx:27]: Can we use just 'ref' property?

Comment (zaporozhetsAnton) [reply src/components/Tags/index.tsx:27]: As I know no. How do you think we can use "ref" for calling Tags method outside, from parent component?

Comment (pavel-stryber) [reply src/components/Tags/index.tsx:27]: Ref actually refers to 'this' of the component. This should work:
```
const tagsRef = React.createRef();
const clearTags = () => {
 tagsRef.current.resetTags();
}

<Tags ref={tagsRef} />

```

Comment (zaporozhetsAnton) [reply src/components/Tags/index.tsx]: You're right, just changed.

**Review by pavel-stryber — APPROVED** [review]

**Pull request merged**